src/main/java/org/reaktivity/nukleus/amqp/internal/stream/AmqpServerFactory.java

```diff
@@ -2982,6 +3010,7 @@ private void cleanup(
                     this.initialId = supplyInitialId.applyAsLong(newRouteId);
                     this.replyId = supplyReplyId.applyAsLong(initialId);
                     this.application = router.supplyReceiver(initialId);
+                    this.detachError = false;
```

Comment:
No need to explicitly assign language-defined default values, like `false` for `boolean`.